blockmanager.go

```diff
@@ -423,19 +457,9 @@ func (b *blockManager) handleDonePeerMsg(peers *list.List, p *peer) {
 
 	bmgrLog.Infof("Lost peer %s", p)
 
-	// Remove requested transactions from the global map so that they will
-	// be fetched from elsewhere next time we get an inv.
-	for k := range p.requestedTxns {
-		delete(b.requestedTxns, k)
-	}
-
-	// Remove requested blocks from the global map so that they will be
-	// fetched from elsewhere next time we get an inv.
-	// TODO(oga) we could possibly here check which peers have these blocks
-	// and request them now to speed things up a little.
-	for k := range p.requestedBlocks {
```

Comment:
This, and the previous for requstedTxns, is still going to be needed.  It will just be done against the peer data now.